web/modules/custom/os2forms_queued_email/src/Plugin/AdvancedQueue/JobType/QueuedEmail.php

```diff
@@ -88,10 +85,15 @@ public function process(Job $job): JobResult {
         // Handle OS2Forms attachments.
         if (isset($attachment[self::OS2FORMS_QUEUED_EMAIL_CONFIG_NAME])) {
           $os2formsAttachmentFilenames[] = $attachment[self::OS2FORMS_QUEUED_EMAIL_CONFIG_NAME];
+
+          if (FALSE === file_exists($attachment[self::OS2FORMS_QUEUED_EMAIL_CONFIG_NAME])) {
```

Comment:
`file_exists` returns a boolean value:

```suggestion
          if (!file_exists($attachment[self::OS2FORMS_QUEUED_EMAIL_CONFIG_NAME])) {
```

Reply:
Handled.